google/cloud/bigtable_v2/services/bigtable/transports/base.py

```diff
@@ -132,37 +132,27 @@ def _prep_wrapped_messages(self, client_info):
         self._wrapped_methods = {
             self.read_rows: gapic_v1.method.wrap_method(
                 self.read_rows,
-                default_timeout=43200.0,
+                default_timeout=None,
```

Comment:
The timeout and retry values were changed automatically in this file after pulling down upstream changes in the gapic/proto files.

We can safely ignore these values for now, since we will be using a layer over top of the base transport. This is more relevant for fully autogenerated libraries